pipelines/lib/util.groovy

```diff
@@ -460,6 +460,9 @@ def void jenkinsWrapper(Map buildParams) {
 
     // This line uses k8s to set EUPSPKG_NJOBS
     def njobs = env.K8S_DIND_LIMITS_CPU
+    if (njobs == null){
```

Comment:
I'm worried about why the value is not available.  Is it not available anywhere, or only in certain cases?

Reply:
Seems to affect Macs who do not have it set.